implementations/rust/ockam/ockam_command/src/rendezvous_server/start/healthcheck.rs

```diff
@@ -0,0 +1,149 @@
+use ockam::transport::HostnamePort;
+use ockam::udp::{RendezvousClient, UdpBindArguments, UdpBindOptions, UdpTransport};
+use ockam_api::DefaultAddress;
+use ockam_core::errcode::{Kind, Origin};
+use ockam_core::{route, AsyncTryClone, Error, Result};
+use ockam_node::Context;
+use std::net::SocketAddr;
+use tokio::io::{AsyncReadExt, AsyncWriteExt};
+use tokio::net::TcpListener;
+use tokio::task::JoinHandle;
+use tracing::info;
+
+pub struct Healthcheck {
```

Comment:
I'm not 100% convinced that this logic should be in this repo... This kind of logic looks more like the stuff we have in `ockam_api`, and the same thing for the errors. We can come back to this later.

We have a similar thing for the node [here](https://github.com/build-trust/ockam/blob/1d63a294ededc1a5b5498f3cd4d486bfef1702e7/implementations/rust/ockam/ockam_api/src/nodes/service/http.rs#L25), as a reference of what I have in mind.

Reply:
I'm OK with moving that to `ockam_api`. Does that work?

Reply:
Yes, I think it would fit better there.